presets/expedition.json

```diff
@@ -0,0 +1,33 @@
+{
+  "metadata": {
+    "id": "expedition",
+    "name": "Expedition",
+    "description": "Start with Soul of Bat, Leap Stone, Gravity Boots, Duplicator, Manna prism, and Buffalo star in a faster, lower-complexity adventure preset.",
+    "author": "3snow_p7im and eldri7ch",
+    "weight": -100
+  },
+  "inherits": "nimble",
+  "stats": false,

```

Comment:
Doesn't Nimble already have stat rando turned off? If so you can remove this line.